include/ck/utility/data_type.hpp

```diff
@@ -990,69 +1006,657 @@ struct vector_type<T, 256>
     }
 };
 
-using int64_t = long;
+template <typename T, index_t N>
+struct non_native_vector_base
+{
+    using VecT = non_native_vector_base<T, N>;
```

Comment:
I'd suggest using `type` name for this alias, as for all other vector types `type` stands for kinda _basic_ or _native_ underlying data type representation... 
```suggestion
    using type = non_native_vector_base<T, N>;
```

Reply:
Updated in [a7b2126](https://github.com/ROCm/composable_kernel/pull/1333/commits/a7b2126acb64b0491a473be157c2d602469bd542)